spark/src/main/kotlin/com/adevinta/spark/components/textfields/SparkTextFieldImpl.kt

```diff
@@ -524,11 +518,10 @@ internal const val AnimationDuration = 150
 private const val PlaceholderAnimationDuration = 83
 private const val PlaceholderAnimationDelayOrDuration = 67
 
-internal val TextFieldPadding = 16.dp
 internal val CounterPadding = 8.dp
-internal val HorizontalIconPadding = 12.dp
+internal val HorizontalIconPadding = 8.dp
```

Comment:
Could we group all Paddings in a different file and maybe inside an object? So that they could be easily found and updated if needed?

Reply:
I prefer not as it's already tedious to find where we need to change the padding and then we would have to go to another file to know whats the padding value without knowing if there'll be side effects as it'll not be highlighted in the file 😞